dataModel.OSLO.PassengerTransportHubs/BicycleParkingStationPTH-AP/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Have you checked this payload in a context broker ? 
Are you sure it is correct?